src/modules/layout/components/LocationFooter.js

```diff
@@ -24,9 +24,10 @@ export class LocationFooter extends React.PureComponent<PropsType> {
       <CleanLink to={new DisclaimerRouteConfig().getRoutePath({ city, language })}>
         {t('imprintAndContact')}
       </CleanLink>
+      <CleanAnchor href='https://integreat-app.de/about/'>{t('settings:about')}</CleanAnchor>
```

Comment:
There is also an english version of the about page, https://integreat-app.de/en/about/ . If the content language is english, it should be displayed.